### PR TITLE
Bundle size

### PR DESCRIPTION
**Pull request by mkholjuraev**
# Description

Associated Jira ticket: # (issue)

This PR disables loading `@patternfly/react-icons`,` @patternfly/react-table` packages, but forces to load the shared deps from the default scope in federated module. With this in place, dependant apps, will not load their local copy of the package, but use the shared one as both a fallback module and main instance in the shared scope. This PR, should be looked at together with this demo Patch PR.

# How to test the PR

Please include steps to test your PR.

# Before the change

# After the change

# Dependent work link

# Checklist:

- [ ] The commit message has the Jira ticket linked
- [ ] PR has a short description
- [ ] Screenshots before and after the change are added
- [ ] Tests for the changes have been added
- [ ] README.md is updated if necessary
- [ ] Needs additional dependent work

### CODE REVIEW

File: src/SmartComponents/Remediation/AsyncRemediationButton.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import React from 'react';
 import { useDispatch } from 'react-redux';
 import AsyncComponent from '@redhat-cloud-services/frontend-components/AsyncComponent';
 import { Spinner } from '@patternfly/react-core';
-import { spinnerSize } from '@patternfly/react-core/dist/js/components/Spinner/Spinner';
+// import { spinnerSize } from '@patternfly/react-core/dist/js/components/Spinner/Spinner';
```
Comment (leSamo) [inline src/SmartComponents/Remediation/AsyncRemediationButton.js:7]: Can we remove this comment for cleanliness?

**Comment by Fewwy** [comment]
I couldn't open patch when I'm running this PR.
The instruction says nothing about this issue so it's a bug?
image

image

**Comment by mkholjuraev** [comment]
@Fewwy fixed!

**Comment by Fewwy** [comment]
Everything works as expected

**Review by Fewwy — APPROVED** [review]

**Pull request merged**